package.json

```diff
@@ -42,7 +42,7 @@
     "json-loader": "^0.5.4",
     "json5": "^0.5.0",
     "jsx-control-statements": "3.1.5",
-    "leaflet": "0.7.3",
+    "leaflet": "1.0.3",
```

Comment:
Do you expect new point releases to cause any problems?  If not, we should use `^1.0.3`.